text/0000-general-fund.md

```diff
@@ -0,0 +1,110 @@
+
+* Activities related to the continuing development needs of the Grin code-base and related projects under the GitHub [mimblewimble](github.com/mimblewimble) organization. This can include:
+   * Fixed-term development or project-management contracts
+   * Infrastructure and other incidental costs
+   * Security audit costs
+   * Costs incurred for any legal matters that may arise for either the project or individual contributors arising directly from their work in good faith on the Grin project
+* Running budgets for Grin teams (with funds spent at their own discretion)
+* Technological research in areas that may be relevant to Grin in the future
+* Cryptographic research related to Grin or Mimblewimble in general
+* Fully open-sourced community-based projects that enhance the Grin ecosystem
+* Supporting non-profit, community-driven and inclusive promotional efforts for the Grin Project (e.g. Grincon) 
+* Charitable donations toward important causes relevant to the Grin or cryptocurrency communities as a whole
+
+While the guidelines above give plenty of flexibility in how funds can be employed toward the advancement of the Grin project, certain types of funding requests will not be considered:
+
+* Marketing or promotional 'offers' where a party is taking a profit or gains some financial benefit to promote Grin (regardless of how the 'opportunity' is presented)
+* Cryptocurrency speculation or any kind of investment
+
+## Request for Funding Process
+
+All requests for funding should be made in the [Governance Section](https://forum.grin.mw/c/governance) of the Grin forum. Topics should be clearly titled "Request for Funding: " followed by a short description of the request.
+
+There is no set template for what the contents of the funding request should include, however a better reception can be attained by including:
+
+* The exact amount requested, what the funding covers and for what time period (if applicable)
+* A clear description of what will be provided to the Grin project in return, including:
+    * Details of any work or research that will be performed during the period
+    * The expected deliverable, if applicable (for a research project, for instance)
+* Background information on the person making the request
+
+Amounts should be stated using fiat currency equivalents with the exact Cryptocurrency amount determined by the rate at the time of transfer. To assist others with conceptualizing the amounts, fiat currencies used should be reasonably global, e.g. USD, EUR, RMB, GBP.
+
+All community discussion on the funding request should be made within the forum thread itself, and the final decision on the request will be made by members of the core team at a public governance meeting, to take place at least one week after the request is made. The community is encouraged to display their support (or lack thereof) of a funding request at these Governance meetings, however the decision rests with the core team.
```

Comment:
add a link for `public governance meeting` pointing to https://github.com/mimblewimble/grin-pm/issues?q=is%3Aissue+label%3Agovernance+label%3Ameetings+